### DISCUSSION TITLE
I can’t set up switching between two cameras

### DISCUSSION THREAD

**Discussion by Suharevsky171296** [iNavFlight/inav#10505] [General]
Please tell me how can I implement switching between two cameras via the sbus pin on the speedybee f405v4 flight controller?

**Comment by Suharevsky171296** [comment]
All servo contacts are busy so I can’t solder them. Switcher I use Vifly Cam Switcher v1.0

**Comment by MrD-RC** [comment]
You can’t. The SBUS pin is just a trace with a hardware inverter between it and a UART RX. I believe UART 2.

**Reply by Suharevsky171296** [reply]
please tell me how then can I implement switching cameras using this switchеr? Where can I connect to configure the switch from the remote control?

**Reply by Suharevsky171296** [reply]
I understand that it is possible via pwm, but where can I solder it so that there is a pwm signal?

**Reply by sensei-hacker** [reply]
If you have a camera switcher that accepts PWM inputs you can set the LED pin as PWM.
This is best done by using INAV 8?0RC2, a pre release version.

**Reply by Suharevsky171296** [reply]
do you mean the output of the LED strip?

**Reply by Suharevsky171296** [reply]
and another question: after I reassign the output of the LED strip to PWM, can I then assign it to the remote control switch?

**Reply by Suharevsky171296** [reply]
If this is possible, please tell me how to implement it? I want to assign it to the switch of the copter control panel since I don’t have a separate OSD remote control

**Reply by sensei-hacker** [reply]
> do you mean the output of the LED strip?

The output which would otherwise be used for the LED strip, yes.

**Reply by sensei-hacker** [reply]
> If this is possible, please tell me how to implement it? I want to assign it to the switch of the copter control panel since I don’t have a separate OSD remote control

You assign it as a "servo".  Then in the mixer you assign whichever RC channel you want to that "servo".

**Reply by Suharevsky171296** [reply]
Thanks a lot. I'll try. I'll write the results. Maybe it will help someone

**Reply by Suharevsky171296** [reply]
> > Если это возможно, то подскажите, как это реализовать? Хочу назначить на переключатель пульта управления коптера, так как отдельного пульта OSD у меня нет
> 
> Вы назначаете его как «серво». Затем в микшере назначаете любой нужный вам канал RC этому «серво».

I don’t have the opportunity to upgrade to version 8, is there any way to do it from version 7.1.0?

**Reply by Suharevsky171296** [reply]
> > Если это возможно, то подскажите, как это реализовать? Хочу назначить на переключатель пульта управления коптера, так как отдельного пульта OSD у меня нет
> 
> Вы назначаете его как «серво». Затем в микшере назначаете любой нужный вам канал RC этому «серво».

thanks for the advice. everything works as you said
